packages/react-sdk/src/components/list-item.ws.tsx

```diff
@@ -15,6 +15,7 @@ const presetStyle = {
 export const meta: WsComponentMeta = {
   category: "typography",
   type: "rich-text",
+  allowedParentComponents: ["List"],
```

Comment:
lets brainstorm the name a little:
-`validParents`
- `allowedParents`
- `acceptedParents`

I tend to `acceptedParents`

Reply:
no strong opinion though